server/src/main/java/com/netflix/conductor/noopindex/NoopEmbeddedElasticSearchProvider.java

```diff
@@ -0,0 +1,16 @@
+package com.netflix.conductor.noopindex;
```

Comment:
Please add license headers to new files.

Reply:
@apanicker-nflx Thanks. I added the license header using `./gradlew licenseFormat`

On the added files, I put my employer's name in the copyright notice since I made this change on work time.

Reply:
I understand your concern about making the changes on work time, however, since the project is open sourced under Netflix, the header would need to say Netflix. Thanks for your contribution.